lectures/getting_started_julia/us_cities.txt

```diff
@@ -1,9 +1,9 @@
 new york: 8244910
-los angeles: 3819702
-chicago: 2707120
-houston: 2145146
-philadelphia: 1536471
-phoenix: 1469471
-san antonio: 1359758
-san diego: 1326179
-dallas: 1223229
\ No newline at end of file
+        los angeles: 3819702
```

Comment:
SHould this be changed?  Maybe a mistaken edit.  I would just get rid of tthe spaces if so